pytorch_forecasting/data/timeseries.py

```diff
@@ -1654,7 +1654,7 @@ def _collate_fn(
                     scale = torch.tensor([batch[0]["target_scale"][idx] for batch in batches], dtype=torch.float)
                 target_scale.append(scale)
         else:  # convert to tensor
-            target_scale = torch.tensor([batch[0]["target_scale"] for batch in batches], dtype=torch.float)
+            target_scale = torch.tensor(np.array([batch[0]["target_scale"] for batch in batches]), dtype=torch.float)
```

Comment:
Could you use `torch.from_numpy`? This should allow it to share memory and speed things up.

Reply:
Done